js/huobi.js

```diff
@@ -2874,6 +2874,14 @@ module.exports = class huobi extends Exchange {
         return result;
     }
 
+    networkPairIdFromCode (currencyCode, networkId) {
```

Comment:
@sc0Vu I think this bug was caused by a typo, it should be `networkCodeToId` instead of `networkPairIdFromCode`

@ttodua can you confirm?

Reply:
uh oh. right, it was typo, should be `networkCodeToId (networkCode, code)`.

Reply:
I am also working on those network related things atm to make more deeper coverage and fixes, and will submit other PR shortly.

Reply:
if you see any issue on github from users, related to network-things, you can tag me there.